backend/inspirehep/mailing/cli.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2019 CERN.
+#
+# inspirehep is free software; you can redistribute it and/or modify it under
+# the terms of the MIT License; see LICENSE file for more details.
+
+
+import click
+import structlog
+from flask.cli import with_appcontext
+
+from .api.jobs import (
+    get_jobs_from_last_week,
+    get_jobs_weekly_html_content,
+    send_jobs_weekly_campaign,
+)
+
+
+@click.group()
+def mailing():
+    """Command to handle mailing."""
+
+
+@mailing.command(help="Sends a campaign with the INSPIRE jobs posted last week.")
+@click.option("-t", "--test", help="Send test campaign.", is_flag=True)
```

Comment:
would it be enough to have only --test-emails? I don't see the point of having both test and emails. If the user provides the emails, then we can automatically assume that he wants to send a test campaign